build.sbt

```diff
@@ -12,7 +12,7 @@ val Slf4jVersion      = "1.7.26"
 
 lazy val scalatest     = "org.scalatest"     %% "scalatest"       % ScalaTestVersion         % Test
 lazy val scalacheck    = "org.scalacheck"    %% "scalacheck"      % "1.15.4"                 % Test
-lazy val scalatestplus = "org.scalatestplus" %% "scalacheck-1-15" % s"${ScalaTestVersion}.0" % Test
+lazy val scalatestplus = "org.scalatestplus" %% "scalacheck-1-15" % s"3.2.11.0" % Test // Cannot upgrade to 3.2.12.0 until Scalacheck 1.16.x
```

Comment:
Hm, what's going on here?

Reply:
Hmm perhaps we don't need to do this - I didn't realise that Scalacheck 1.16.x was binary compatible with 1.15.x

Reply:
Scalacheck 1.16.x should be binary and source compatible with 1.15.x. So it should all work.